pynvme/README.md

```diff
@@ -5,28 +5,37 @@ We use [SWIG](http://www.swig.org/) to generate Python bindings for libnvme.
 ## How to use
 
 ```python
+#!/usr/bin/env python3
 import sys
+import pprint
 from libnvme import nvme
 
-root = nvme.root()     # This is a singleton
+root = nvme.root()      # This is a singleton
+root.log_level('debug') # Optional: extra debug info
+
 host = nvme.host(root) # This "may be" a singleton. 
-ctrl = nvme.ctrl(subsysnqn=<nqn>, transport=<trtype>, traddr=<traddr>, trsvcid=<trsvcid>, host_traddr=<traddr or None>, host_iface=<iface or None>)
+ctrl = nvme.ctrl(subsysnqn='nqn.2014-08.org.nvmexpress.discovery', transport='tcp', traddr='fd2e:853b:3cad:e135:506a:65ee:29f2:1b18', trsvcid='8009', host_iface='ens256')
```

Comment:
I am not really sure if it's good to replace the templates with real values. Maybe the adding a comment adding some info about the API would be helpful?

Reply:
Addressed your concerns by removing real values and adding comments